src/solver/simulation/sim_extern_variables_globales.h

```diff
@@ -31,7 +31,6 @@
 #include "sim_structure_probleme_economique.h"
 
 /* Valeurs generees de maniere aleatoire */
-extern std::vector<std::vector<NUMERO_CHRONIQUES_TIREES_PAR_INTERCONNEXION>> NumeroChroniquesTireesParInterconnexion;
```

Comment:
Can we also remove the definition of the struct `NUMERO_CHRONIQUES_TIREES_PAR_INTERCONNEXION` ?